ddl/oracle/upgrades/1-add-change-passwd.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE SEARCHAPP.SEARCH_AUTH_USER ADD CHANGE_PASSWD NUMBER(1,0);
```

Comment:
Better to use 001 so they can be ordered in the usual lexicographical order where codepoints are compared one by one (like in Java).